lambdas/indexer/app.py

```diff
@@ -60,11 +60,22 @@ def basic_health():
 
 
 @app.route('/health', methods=['GET'], cors=True)
-def health():
+@app.route('/health/{key}', methods=['GET'], cors=True)
+def health(key: Optional[str]=None):
     health = Health('indexer')
+    body = health.as_json(*[(key,)] if key else [])
     return Response(
-        body=json.dumps(health.as_json),
-        status_code=200 if health.up else 503
+        body=json.dumps(body),
+        status_code=200 if body['up'] else 503
+    )
+
+
+@app.route('/progress', methods=['GET'], cors=True)
```

Comment:
We don't really need this since it returns the same info as `/health/progress`, correct?